django/db/models/query.py

```diff
@@ -678,8 +678,17 @@ def raw(self, raw_query, params=None, translations=None, using=None):
             using = self.db
         return RawQuerySet(raw_query, model=self.model, params=params, translations=translations, using=using)
 
-    def _values(self, *fields):
+    def _values(self, *fields, **expressions):
         clone = self._clone()
+
+        if expressions:
+            def alias(expression):
+                if hasattr(expression, 'resolve_expression'):
+                    return expression
+                return F(expression)
+            expressions = {name: alias(expr) for name, expr in expressions.items()}
```

Comment:
I think an explicit loop that mutates the dict might be clearer here, and avoid the overhead of a function creation and call. I believe we already do this elsewhere (but I haven't double checked).